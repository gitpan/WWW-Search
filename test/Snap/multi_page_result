http://www.snap.com/search/power/results/1,180,home-0,00.html?NR=100&KW=%2Bdiv7+%2BISI&AT0=w&FD=1&XT=&AN=1&DM=&FM=1&DR=0&FR=f&KM=a&LN=&PL=a&AK0=&AM0=m&
http://www.snap.com/search/power/results/1,180,home-0,00.html?/search/power/results/1,180,home-0,00.html?KM=a&KW=%2Bdiv7+%2BISI&AM0=m&AT0=w&AK0=&AM1=&AT1=&AK1=&AM2=&AT2=&AK2=&AM3=&AT3=&AK3=&AM4=&AT4=&AK4=&AN=1&NR=100&FR=f&PL=a&DR=0&FM=1&FD=1&FY=&TM=&TD=&TY=&LN=&MT=&XT=&DM=&SDI=&start=100&tag=st.sn.sr.6.pgNR=100&KW=%2Bdiv7+%2BISI&AT0=w&FD=1&XT=&AN=1&DM=&FM=1&DR=0&FR=f&KM=a&LN=&PL=a&AK0=&AM0=m&
