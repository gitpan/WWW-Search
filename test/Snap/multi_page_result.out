http://www.isi.edu/div7/people/postel.home/
http://www.isi.edu/div7/people/cooper.home/
http://www.isi.edu/div7/people/mfanning.home/
http://www.isi.edu/~davidk/ride/
http://www.merit.edu/mail.archives/html/nanog/msg00906.html
http://www.isi.edu/~davidk/6bone/
http://www.isi.edu/~davidk/ride/ietf/38/index.html
http://www.isi.edu/~davidk/ride/ietf/39/index.html
http://nis.nsf.net/mail.archives/html/iepg/msg00219.html
http://www.isi.edu/div7/research-top.html
http://nis.nsf.net/mail.archives/html/iepg/msg00198.html
http://www.saia.asn.au/saix/
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00128.html
http://wais.ripe.net/mail-archives/eof-list/19951001-19951107/msg00005.html
http://wais.ripe.net/mail-archives/eof-list/19951001-19951107/msg00006.html
http://www.cctec.com/maillists/nanog/historical/9610/msg00404.html
http://www.cctec.com/maillists/nanog/historical/9610/msg00409.html
http://www.isi.edu/div7/people/postel.home/activities.html
http://www.merit.edu/mail.archives/html/nanog/msg01520.html
http://www.isi.edu/ra/rps/transition/whois-ripe181.html
http://www.isi.edu/~davidk/6bone/presentations.html
http://www.cctec.com/maillists/nanog/historical/9610/msg00849.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00201.html
http://wais.ripe.net/mail-archives/eof-list/19961101-19961201/msg00004.html
http://wais.ripe.net/mail-archives/eof-list/19960101-19960201/msg00001.html
http://www.cctec.com/maillists/nanog/historical/9511/msg00135.html
http://www.cctec.com/maillists/nanog/historical/9511/msg00197.html
http://nis.nsf.net/mail.archives/html/iepg/msg00004.html
http://wais.ripe.net/mail-archives/eof-list/19970201-19970301/msg00000.html
http://wais.ripe.net/mail-archives/eof-list/19951001-19951107/msg00003.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00264.html
http://www.isi.edu/div7/ln/los-nettos-story.html
http://www.isi.edu/~davidk/ride/presentations.html
http://www.cctec.com/maillists/nanog/historical/9511/msg00368.html
http://www.isi.edu/~joseph/addr.html
http://www.isi.edu/div7/people/postel.home/pictures.html
http://www.gated.merit.edu/~gated/new_web/support/email/old_mail_archives/gated-people/0290.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00231.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00160.html
http://wais.ripe.net/mail-archives/rr-impl/19951001-19951107/msg00001.html
http://www.academ.com/nanog/feb1996/pier.ip.address.survey.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00273.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00274.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00167.html
http://www.merit.edu/mail.archives/html/nanog/msg01526.html
http://www.cctec.com/maillists/nanog/historical/9511/msg00200.html
http://www.cctec.com/maillists/nanog/historical/9610/msg00840.html
http://www.uwsg.indiana.edu/hypermail/linux/kernel/9610.2/0137.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00277.html
http://www.gated.merit.edu/~gated/new_web/support/email/old_mail_archives/gated-people/0329.html
http://telemat.die.unifi.it/book/1997/RSVP/ELABORATO/man_bibliografia.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00200.html
http://www.cctec.com/maillists/nanog/historical/9511/msg00201.html
http://www.iana.org/comments-mail/archives/25july1998-31july1998/msg00019.html
http://www.cctec.com/maillists/nanog/historical/9504/msg00113.html
http://www.netsys.com/firewalls/firewalls-9603/0019.html
http://wais.ripe.net/mail-archives/rr-impl/19951201-19960101/msg00000.html
http://www.cctec.com/maillists/nanog/historical/9511/msg00383.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00268.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00280.html
http://www.isi.edu/div7/cities/tools/autosearch/load_balancing/19961209.html
http://www.isi.edu/lsam/tools/autosearch/load_balancing/19961209.html
http://www.uwsg.indiana.edu/hypermail/linux/kernel/9611/0040.html
http://www.isi.edu/div7/people/jkrey.home/bio.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00104.html
http://wais.ripe.net/mail-archives/rr-impl/19950216-19950801/msg00132.html
http://www.cctec.com/maillists/nanog/historical/9604/msg00168.html
http://www-leland.stanford.edu/~yph/domains/resources/resources.html
http://www-cse.stanford.edu/classes/cs201/projects-97-98/domain-names/resources/resources.html
http://cwis.usc.edu/dept/cs/whats_cool/jobs.html
http://www.usc.edu/dept/cs/whats_cool/jobs.html
http://www.isi.edu/~rdv/netstation/netstation.html~
http://www.apng.org/apcctld/docs/postel-call.html
http://www.apng.org/aptld/docs/postel-call.html
http://prope.insa-lyon.fr/doc/doc-rfc/fromweb-best.txt
http://www.cs.utk.edu/~moore/san-jose/wg/uswg/uswg-minutes-95jul.txt
http://ftp.sunet.se/pub4/Internet-documents/ietf/uswg/uswg-minutes-95jul.txt
http://oddbjorn.bdc.no/mirrors/ftp.ietf.org/uswg/uswg-minutes-95jul.txt
http://www.acm.org/sigcomm/sigcomm95/workshop/attendee.html
http://www.roxen.com/rfc/rfc1879.html
http://www.ictp.trieste.it/~radionet/nuc1996/ref/rfc/rfc1879.htm
http://pyx.eng.ku.ac.th/rfc/rfc1879.html
http://www2.zf.jcu.cz/misc/docs/rfc/rfc1879.html
http://www.kisco.co.kr/RFC/rfc/rfc1879.html
http://home.zf.jcu.cz/asc/misc/docs/rfc/rfc1879.html
http://www.csl.sony.co.jp/rfc/cache/rfc1879.txt.html
http://www.ipa.net/~dustin/rfc/rfc1879.shtml
http://www.aware.fr/conferences/T%8El%8Echargement%20Public/G%8En%8Eral/Normes%20Internet/RFC/rfc18xx/rfc1879
http://myass.com/nic/rfcs/1800/rfc1879.txt.html
http://www.uic.edu/~mlanski/routeserver.html
http://gost.isi.edu/~johnh/SOFTWARE/APACHE/apache_1.0_isi_mmap_960524.patch
http://ftp.sunet.se/pub2/Internet-documents/ietf/pier/pier-minutes-96mar.txt
http://www.cs-ipv6.lancs.ac.uk/ipv6/documents/standards/general-comms/ietf/pier/pier-minutes-96mar.txt
http://www.enteract.com/~lspitz/whois.html
http://wais.ripe.net/docs/ripe-178.html
http://linux-howto.com/rfc/rfc1500-1999/rfc1879.txt
http://consult.stanford.edu/pub/internet/netinfo/i-d/draft-manning-classa-exp-00.txt
http://aldebaran.na-cp.rnp.br/rfc/rfc1879.txt
http://laplace.snu.ac.kr/~pos7hink/java-network-programming/rfc/rfc1879.txt
http://ftp.vse.cz/pub/docs/rfc/rfc1879.txt
